scripts/HUMAnN2_pipeline.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+#$ -o ./log/
+#$ -e ./log/
+
+
+#STEPS
+# run the humann2 
+# renormalize to copies per million and relative abundance 
+# map to KO and EC terms
+# remove the intermediate files
+
+#specify paths 
+
+WORKING_DIR=/klaster/scratch/mmaranga/functional_analysis/diabimmune_data/
+READ_DIR=${WORKING_DIR}/combined_data 
+OUTPUT_DIR=${WORKING_DIR}/uniref90_diabimmune_humman2_analysis 
+MAPPING_FILES=/home/mmaranga/utility_mapping/
+
+cd $WORKING_DIR
+
+# running humann2
```

Comment:
how long does the pipeline take? can I run it on a single computer or do I need to run it on a cluser? If it needs to be run on a cluster, please add a docstring at the top specifying that and include sample parameters for a specific queuing system. If the script can be run on a single computer, but it takes more than a few minutes, I'd consider changing the comments (like `#running humann2`) into print statements (`echo "running humann2...`), so that the user knows the script is executing.

Reply:
Okay will make changes on that. Diamond translated search part is very computational intensive. Takes like a day for one sample.  Would't be advisable to run the pipeline on a personal computer. Especially if one has multiple files